### ISSUE TITLE
Qubes Cannot See disk drives on Dell XPS 15 9550

### ISSUE THREAD

**Issue by Drako365** [QubesOS/qubes-issues#2448]
Qubes-R3.2-x86_64

I booted up the installation ISO (in legacy and UEFI modes) using a USB flash drive on a Dell XPS 15 9550. I navigated through the language selection, and when I reached the installation summary screen, it said that in the Installation Destination there were no disks selected. I clicked that, and was greeted with an empty disk list, and a suggestion to, "...shut down the computer, connect at least one disk, and restart to complete installation."

I have tried every promising setting in the BIOS, enable/disable secure boot, enable/disable thunderbolt USB boot, boot using legacy and UEFI, and even switching SATA operation from RAID to ahcp. I upgraded/downgraded the BIOS, tested other linux distros incase Qubes was somehow in error, and even (as a desperate measure of last resort), called Dell to see if they happened to have any useful information (they didn't, but then, they wouldn't know it if it bit them in the ***). 

Long story short, nothing will get Qubes to even see my hard drives. So I've come straight to the source, and I'm asking. Any idea how to work around this? 

#### Related issues:
#794 
https://groups.google.com/forum/m/#!topic/qubes-users/3kl1LQls_HA

**Comment by andrewdavidwong** [comment]
This question is too localized for `qubes-issues`. Please consider posting this to the `qubes-users` mailing list instead. (You can read more about our mailing lists [here](https://www.qubes-os.org/mailing-lists/).)

`qubes-users` is intended for these sorts of questions and receives much more traffic, which means that your question is more likely to receive a response there.

**Comment by Drako365** [comment]
No problem, thanks for the redirect.
